content/applications/finance/accounting/bank/feeds/bank_statements.rst

```diff
@@ -26,47 +23,46 @@ Odoo supports multiple file formats to import bank statements:
 - Quicken Interchange Format (.QIF)
 - Belgium: Coded Statement of Account (.CODA)
 
-To import them, go to :menuselection:`Accounting --> Overview --> Bank`, click on *Import
-Statements*, or on the three dots, and then on *Import Statement*.
+To import a file, go to :menuselection:`Accounting --> Overview --> Bank`, click on
+:guilabel:`Import Statements`, or on the ⋮ button and then on :guilabel:`Import Statement`.
```

Comment:
Is it necessary to mention the different paths..? I think just mentioning the Import Statement option available from the Dashboard is enough but that's a minor comment. Up to you :)

Reply:
Just so people know, and this way they also discover/know about the other options available by clicking on the vertical ellipsis :)